script/zmb

```diff
@@ -613,6 +613,58 @@ sub cmd_get_batch_job_result {
 }
 
 
+=head2 batch_status
+
+ zmb [GLOBAL OPTIONS] batch_status [OPTIONS]
+
+ Options:
+   --batch-id BATCH-ID|--bi BATCH-ID
+   --list-waiting-tests|--lw
+   --list-running-tests|--lr
+   --list-finished-tests|--lf
+
+"--batch-id" is mandatory.
+
+The command provides the number of tests waiting to be run, tests running and
+test finished, respectively, for the batch.
+
+"--list-waiting-tests", "--list-running-tests" and "--list-finished-tests" are
+optional. If given the test IDs of tests waiting to be run, tests running
+and test finished, respectively, are listed.
+
+=cut
+
+sub cmd_batch_status {
+    my @opts = @_;
+
+    my $opt_batch_id;
+    my $opt_list_waiting_tests;
+    my $opt_list_running_tests;
+    my $opt_list_finished_tests;
+
+    GetOptionsFromArray(
+        \@opts,
+        'batch-id|bi=s'        => \$opt_batch_id,
+        'list-waiting-tests|lw'  => \$opt_list_waiting_tests,
+        'list-running-tests|lr'  => \$opt_list_running_tests,
+        'list-finished-tests|lf' => \$opt_list_finished_tests,
+    ) or pod2usage( 2 );
+
+    my %params;
+    $params{batch_id} = $opt_batch_id;
+
+    $params{list_waiting_tests} = "true" if $opt_list_waiting_tests;
+    $params{list_running_tests} = "true" if $opt_list_running_tests;
+    $params{list_finished_tests} = "true" if $opt_list_finished_tests;
```

Comment:
Maybe sent proper booleans instead of strings?
```suggestion
    $params{list_waiting_tests} = JSON->boolean( $opt_list_waiting_tests ) if defined $opt_list_waiting_tests;
    $params{list_running_tests} = JSON->boolean( $opt_list_running_tests ) if defined $opt_list_running_tests;
    $params{list_finished_tests} = JSON->boolean( $opt_list_finished_tests ) if defined $opt_list_finished_tests;
```

Reply:
I haven't actually tried it, but I'm referring to this guy: https://metacpan.org/pod/JSON#boolean

Alternatively, this is another option: https://metacpan.org/pod/JSON#JSON::true,-JSON::false,-JSON::null

Reply:
I found another way using the `json_tern()` function in the script.